frontend/src/app/about/page.tsx

```diff
@@ -159,20 +159,38 @@ const About = () => {
         </SecondaryCard>
 
         <SecondaryCard icon={faMapSigns} title={<AnchorTitle title="Roadmap" />}>
-          <ul>
-            {roadmap.map((item) => (
-              <li key={item.title} className="mb-4 flex flex-row items-center gap-2 pl-4 md:pl-6">
-                <div className="h-2 w-2 flex-shrink-0 rounded-full bg-gray-600 dark:bg-gray-300"></div>
-                <Link
-                  href={item.issueLink}
-                  target="_blank"
-                  className="text-gray-600 hover:underline dark:text-gray-300"
-                >
-                  {item.title}
-                </Link>
-              </li>
+          <div className="grid gap-4">
+            {projectMetadata.recentMilestones.map((milestone, index) => (
+              <div
+                key={index}
+                className={`flex items-center gap-4 overflow-hidden rounded-lg bg-gray-200 p-6 dark:bg-gray-700 ${
+                  index % 2 === 0 ? '' : 'ml-8'
```

Comment:
Can we update the layout to always have the same alignment for all items? I understand the idea behind it, but for now I'm thinking maybe we could simplify this. 
![Screenshot 2025-05-24 at 12 32 07 PM](https://github.com/user-attachments/assets/2c671108-c8fd-4b92-97b4-184862468f56)